stl/src/StlLCMapStringW.cpp

```diff
@@ -1,9 +1,6 @@
 // Copyright (c) Microsoft Corporation.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-#include <internal_shared.h>
-#include <locale.h>
-

```

Comment:
This file should include `<cstring>` because it calls `wcsnlen` (I am not sure how it is being dragged in via `awint.hpp`).